src/Microsoft.AspNetCore.Antiforgery/ServiceCollectionExtensions.cs

```diff
@@ -10,9 +10,16 @@
 
 namespace Microsoft.Extensions.DependencyInjection
 {
+    /// <summary>
+    /// Extension methods for setting up antiforgery services in an <see cref="IServiceCollection" />.
+    /// </summary>
     public static class ServiceCollectionExtensions
     {
-        public static IServiceCollection AddAntiforgery(this IServiceCollection services)
+        /// <summary>
+        /// Adds antiforgery services to the specified <see cref="IServiceCollection" />.
```

Comment:
I think the comment is fine as it is. I think any reader would perfectly well understand what this does.